.changeset/canonical-fields-omnigraph.md

```
@@ -0,0 +1,5 @@
+---
+"ensapi": minor
+---
+
+**Omnigraph**: expose `Domain.canonical` and `Registry.canonical` on the Omnigraph schema. Both are nullable `Boolean` fields indicating whether the entity participates in the canonical namegraph.
```

markdownlint-cli2 (0.22.1)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)